src/dos/dos_programs.cpp

```diff
@@ -1764,7 +1798,7 @@ void DOS_SetupPrograms(void) {
 	        "  \033[36;1mBOOTIMAGE\033[0m is a bootable disk image with specified -size GEOMETRY:\n"
 	        "            bytes-per-sector,sectors-per-head,heads,cylinders\n"
 	        "Notes:\n"
-	        "  - Image paths and filenames are case-sensitive and either relative or\n"
+	        "  - Image paths and filenames are case-insensitive and either relative or\n"
```

Comment:
Regarding the home directory checks, `Cross::ResolveHomedir(home_resolve)`, can the image file also be relative to this home directory?

Reply:
No, only relative to current working dir or virtual DOS C: drive. `Cross::ResolveHomedir` (I don't like this function) only converts leading `~` or `~name` into home dir. I duplicated this functionality in `to_native_path` by using `glob` specific flags.

Reply:
Ahh; thanks for explaining this. Yes, `ResolveHomedir` is indeed misleading because it implies that it's going to get the user's home directory.  I suppose it should be `ReplaceTildeWithHome`, especially if it's a Linux-only function.